driver/riverqueue-sequel/lib/driver.rb

```diff
@@ -76,6 +59,7 @@ def transaction(&)
     private def insert_params_to_hash(insert_params)
       # the call to `#compact` is important so that we remove nils and table
       # default values get picked up instead
+      # TODO: but I had to remove it for bulk unique inserts...
```

Comment:
Wanted to run this one buy you before deleting the comment. When using `multi_insert`, all hashes in the array must have the same keys, because the keys from the first item are used to determine which columns to insert. This is obviously not compatible with using `.compact` to remove keys with `nil` values here, because that would give each row potentially different keys.

As it turns out though, the only one of these inserted fields that actually rely on DB defaults is the `tags` field. I added a client-level default for it, which means (a) the `: nil` below isn't needed, and (b) I think we can safely use this bulk insert mechanism for all inserts.

Thoughts before I clear out those remnants?

Reply:
Yeah seems okay, I think the `#compact` really was needed to make things work properly (although can't remember the specific details), but if adding `tags` does it and the tests pass, I guess we're good.